tools/inventory-extract-windows/main.go

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2021 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/diskfs/go-diskfs/filesystem/fat32"
+	"github.com/diskfs/go-diskfs/partition"
+)
+
+var dirFlag = flag.String("dir", "", "save file to provided directory (will generate temp if empty)")
```

Comment:
NIT: I would name the variable `outputPath`. `dirFlag` does not give me any hint what does the variable mean

Reply:
Fixed